addons/website_event_track/models/event_track_stage.py

```diff
@@ -19,14 +19,11 @@ class TrackStage(models.Model):
     fold = fields.Boolean(
         string='Folded in Kanban',
         help='This stage is folded in the kanban view when there are no records in that stage to display.')
-    is_accepted = fields.Boolean(
-        string='Accepted Stage',
-        help='Accepted tracks are displayed in agenda views but not accessible.')
-    is_done = fields.Boolean(
-        string='Done Stage',
-        help='Done tracks are automatically published so that they are available in frontend.')
+    is_visible_in_agenda = fields.Boolean(string='Visible in agenda',
```

Comment:
In the end this is just a renaming ? I stil prefer is_accepted as it indicates we accepted this proposal. We can change the string though.